backend/cmd/cmd.go

```diff
@@ -41,6 +43,23 @@ func NewRootCmd(
 		"migration migrations root directory",
 	)
 
+	idCmd := cmdFactory.NewCommand(cli.CommandConfig{
+		Usage:        "data-id",
+		ShortHelpMsg: "Use user ID to uniquely identify a user",
+		OnExecute: func(cmd *cli.Command, args []string) {
```

Comment:
cli.Command is interface therefore a pointer itself

Reply:
@arberiii Yes?

Reply:
I thought we don’t to pass the pointer, like just cli.Command. What do you think?

Reply:
@arberiii Adde TODO to update the signature in the app framework.